### ISSUE TITLE
examples/api_test.py needs fixing

### ISSUE THREAD

**Issue by ivuk** [lxc/pylxd#24]
In 5b7895d15e43e7c410ef248f34a89e3ff192f216 `examples/api_test.py` got pretty substantial changes, which turned it from a Python 2.x example to some kind of a py2/py3 combination; the shebang is set to `#!/usr/bin/python`, which refers to Python 2 (at least on 15.04 and earlier - granted, this might depend on the `virtualenv` usage) and, as far as I can tell, it seems that the 'strict' parameter from the [`http_client` call in `pylxd/connection.py`](https://github.com/lxc/pylxd/blob/master/pylxd/connection.py#L30) has been [removed in Python 3.4](https://docs.python.org/3/library/http.client.html), so running the example on 15.04 fails with:
```
TypeError: __init__() got an unexpected keyword argument 'strict'
```
Note that I'm omitting the full output here.

This seem to be easy things to fix, I can try to clean this up a bit if needed.

**Comment by javacruft** [comment]
Hey ivuk

Thanks for the feedback - if you'd like to take a run at improving the example code that would be awesome.

I'm intending on landing a neat concise example into the README - happy for it to be yours!

**Comment by ivuk** [comment]
@javacruft, sorry for not responding sooner, it seems work caught up with me for a bit. :)

Here's what I'd propose as the next step:
- Cleaning up of the `examples/api_test.py` so it looks like "proper" Python 2.x code, i.e. reverting it to the  3d86295a3c0058ac9f41dd3db1dcf1aa3b0cb4fd commit and cleaning it up properly - from the `#!` to typos, comments and similar
- Writing a functionally identical Python 3.x example, called i.e. `examples/api_test_3.py` or something like that

If something like this would be acceptable, I'd start working on the fixing the existing `api_test.py`.

**Comment by rockstar** [comment]
Fixed in #86
